demo/index.html

```diff
@@ -6,23 +6,23 @@
     <title>Vite + React + TS</title>
     <script
       type="text/javascript"
-      src="https://cdn.jsdelivr.net/npm/hls.js@latest/dist/hls.min.js"
+      src="https://cdn.jsdelivr.net/npm/hls.js@1.5.6/dist/hls.min.js"
```

Comment:
Can we make all the version stick only to the major changes like 
```suggestion
      src="https://cdn.jsdelivr.net/npm/hls.js@~1/dist/hls.min.js"
```